addon/lib/metrics.js

```diff
@@ -185,7 +156,7 @@ const Metrics = module.exports = {
     };
 
     TelemetryController.submitExternalPing(
-      pingType, payload,
```

Comment:
It won't let me comment on the line, but right above here you have "version: 1" -- it's supposed to be the version of the add-on.  Is that something we can get from our end, or does the add-on have to pass that up to us?